pyccel/codegen/printing/ccode.py

```diff
@@ -344,6 +346,8 @@ def is_c_pointer(self, a):
             return True
         if isinstance(a, FunctionCall):
             a = a.funcdef.results[0].var
+        if hasattr(a, 'dtype') and isinstance(a.dtype, CustomDataType) and a.is_argument:
```

Comment:
The function `getattr` has a default argument that you can use to shorten this line
```suggestion
        if isinstance(getattr(a, 'dtype', None), CustomDataType) and a.is_argument:
```

Reply:
Fixed